Src/BootCamp.Chapter/Credentials.cs

```diff
@@ -0,0 +1,103 @@
+﻿using System;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class Credentials
+    {
+        private const string Separator = ",";
+
+        public string CredentialsFile { get; } = "credentials.db";
+
+        public Credentials()
+        {
+        }
+
+        public Credentials(string credentialsFile)
+        {
+            CredentialsFile = credentialsFile ?? throw new ArgumentNullException(nameof(credentialsFile));
+        }
+
+        private bool FindUser(User user)
+        {
+            string line;
+            bool found = false;
```

Comment:
Keep it consistent. Either explicit all or var all except for cases where it isn't obvious. I think this should be var.

Reply:
I was fixing/updating the code as you were reviewing my PR. I did not know that you will also review it. Thank you for the observation.